genenetwork/genenetwork2
# Reaper bootstrap fix

#### Description
The positioning of the bootstrap results was wrong due to them not accounting for the pseudo-markers added when interval mapping. This was fixed, as well as a separate issue where setting bootstrap or permutations to 0 caused an error (there's a part of the code that converts the list of output files - gwa results, permutation results, and bootstrap results - into a string by joining it, and if one of those files was None it threw an error).

#### How should this be tested?
Do any mapping using qtlreaper and bootstrapping and check that the yellow bootstrap bars align with the QTLs. For the second error, set bootstrap or permutations to 0 and do qtlreaper mapping.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)

#### Questions
